www/front_src/src/Resources/Actions/Resource/Acknowledge/Dialog.tsx

```diff
@@ -78,42 +86,39 @@ const DialogAcknowledge = ({
             helperText={errors?.comment}
           />
         </Grid>
-        <Grid container item direction="column">
-          <Grid item container xs alignItems="center">
-            <Grid item xs={1}>
+        <Grid item>
+          <FormControlLabel
+            control={
               <Checkbox
+                checked={values.notify}
                 inputProps={{ 'aria-label': labelNotify }}
                 color="primary"
-                value={values.notify}
                 onChange={handleChange('notify')}
+                size="small"
```

Comment:
I think we have enough space within the Acknowledge and Downtime dialog to display the checkboxes in their original sizes (it adds some comfort as well). 

```suggestion
```

Reply:
are we sure we want to keep normal size ? cause we already use TextField as "small" in those dialogs